kernels/ZendEngine2/string.c

```diff
@@ -305,19 +311,25 @@ void zephir_camelize(zval *return_value, const zval *str) {
 }
 
 /**
- * Convert a camelized to a dash/underscored texts
+ * Convert a camelized to a dash/underscored texts (an optional delimiter can be specified)
  */
-void zephir_uncamelize(zval *return_value, const zval *str) {
+void zephir_uncamelize(zval *return_value, const zval *str, const zval *delimiter) {
 
 	unsigned int i;
 	smart_str uncamelize_str = {0};
-	char *marker, ch;
+	char *marker, ch, delim;
 
 	if (Z_TYPE_P(str) != IS_STRING) {
```

Comment:
looks good so far, I think it could make sense to add ```Z_STRLEN_P(str)>1``` here aswell.
(to catch invalid input)

I would recommend holding back this commit until master is fixed again,
then you can use that minor change to trigger the CI builds ^^
(which very likely will pass and make this mergeable)